src/main/java/com/google/sps/data/Election.java

```diff
@@ -116,9 +130,52 @@ public Election fromVoterInfoQuery(DatastoreService datastore, JSONObject voterI
       }
     }
 
+    if (voterInfoQueryData.has("earlyVoteSites")) {
+      JSONArray earlyVoteSiteData = voterInfoQueryData.getJSONArray("earlyVoteSites");
```

Comment:
We should put these variables in to constants at some point.

Reply:
I can do this right now. I'm thinking of making a constant array consisting of these three types and iterating through that to avoid the code repetition present right now.

Reply:
@rogerlaucorp I put a TODO in my code but per our discussion in the blue doc, I was thinking of using an enum to store these constant values. I know @casprice used static final Strings to hold constant values, but afaik Java can handle enum constants with String values (https://howtodoinjava.com/java/enum/java-enum-string-example/) and therefore this could be a neat way to handle it. My only concern is if enums are compliant with Datastore, which I am currently looking into now.

Reply:
TODO is fine. If they're not support in datastore (I suspect they're not) you can store an int or string, and then just parse it when you read from datastore.